caddyhttp/fastcgi/fastcgi_test.go

```diff
@@ -51,6 +52,106 @@ func TestServeHTTP(t *testing.T) {
 	}
 }
 
+// listenerWithConnectionCounter adds a counter to keep track
+// of the number of accepted connections.
+type listenerWithConnectionCounter struct {
+	net.Listener
+	sync.RWMutex
+	ConnectionsAcceptedCounter int
```

Comment:
Prefer short names. `counter` is enough here and you can add a line comment if you want.